### DISCUSSION TITLE
pydantic version upper pin 1.7.0

### DISCUSSION THREAD

**Discussion by RyoMazda** [explosion/spaCy#7373] [Help: Installation] [answered]
This issue is about [this line (pydantic version in requirements.txt)](https://github.com/explosion/spaCy/blob/v3.0.0rc2/requirements.txt#L17) that was introduced at [this PR](https://github.com/explosion/spaCy/pull/6308), since I cannot find any related open issue.

I'm happy if I can use newer version of pydantic with spacy=>3.0.0. Thanks.

**Comment by svlandeg** [comment, marked as answer]
For the upcoming v3.0.0rc3, the Pydantic pin has already been updated to [`>=1.7.1,<1.8.0`](https://github.com/explosion/spaCy/blob/develop/requirements.txt#L17). Meanwhile, if you want to use v3.0.0rc2 with a higher Pydantic version, it probably works from 1.7.1 onwards, so you can update your local installation.

**Comment by RyoMazda** [comment]
Thank you for your kind response. I should have checked the `develop` branch to see what's going on in the upcoming version.

**Comment by svlandeg** [comment]
No worries! It's not always easy to keep track of the different branches :-)
I'll go ahead and close this, assuming your question was resolved ;-)
